### ISSUE TITLE
SPIKE - Data Watcher Database Content

### ISSUE THREAD

**Issue by tech3371** [IMAP-Science-Operations-Center/sds-data-manager#105]
## Description

Short description of the issue you would like to solve. 

## Requirements
- Requirement 1
- Requirement 2

### Nice to have or Goal Requirements
- Nice to have 1
- Nice to have 2

## Additional notes
Any additional notes on the issue, if you have them. 

## Related tickets
- `#123` 

## Follow up tickets
This spike is not considered complete until at least one follow up issue is created. 
- [ ] Placeholder ticket

**Below is the template for the response to this ticket. Add as many solutions as needed, but preferably include 2-5 for discussion. The response should be posted as a comment on this issue, or linked in a comment. **

# Solution 1
Write an overview of the solution here.

## Pros:
- pro 1

## Cons:
- con 1

## Additional notes:
- note

# Solution 2
Write an overview of the solution here.

## Pros:
- pro 1

## Cons:
- con 1

## Additional notes:
- note

# Summary
Write up a summary of your findings, including your preferred solution.

**Comment by tech3371** [comment]
Link to conversation thread that sparked this discussion: https://github.com/IMAP-Science-Operations-Center/sds-data-manager/pull/104#discussion_r1271465224

**Comment by maxinelasp** [comment]
Blocked on #95

**Comment by maxinelasp** [comment]
Per our discussions, we have determined that this information will be made available in the metadata database. As long as we can filter by instrument level, version, etc as laid out in [this document](https://lasp.colorado.edu/galaxy/pages/viewpage.action?spaceKey=IMAP&title=User+Stories) we should have all the information we need to determine if processing is ready.

**Comment by maxinelasp** [comment]
What is left to figure out: exactly what the information we need from the metadata table to determine if processing can proceed, and if we need some additional data tracking tools for internal or external use (eg an api which can return the status of processing, given an instrument, level, and timestamp)

**Comment by tech3371** [comment]
For this SPIKE, I was hoping that we could decide what the database column will look like.
instrument | filename | data_level | version | status 
-- | -- | -- | -- | -- | 

But this might come together during implementation based on user stories and how data watcher is implemented. 
So we can close this SPIKE.

**Comment by maxinelasp** [comment]
Completed with #95, follow up #185
